Dockerfile

```diff
@@ -1,17 +1,11 @@
-FROM maven:3.5.0-jdk-8-alpine
-EXPOSE 8080
+FROM jetty:9.4-jre8
 
-WORKDIR /tmp/src/
```

Comment:
This Dockerfile assumes that application has been already built. It's not fully correct - please check link which I provided in the ticket: idea is to build Orienteer in one container, but then copy war into another one with jetty - which will be the final one.